bulletroboy/exoforce_simulation.py

```diff
@@ -18,7 +19,8 @@ def __init__(self, cage_conf, human_model, mode):
 
 		self.mode = mode
 		self.operator = Operator(human_model)
-
+		self.operator_publisher = self.create_publisher(PoseStamped, '/roboy/simulation/operator_info', 10)
```

Comment:
use topic name - /roboy/simulation/operator/pose/endeffector as i wrote in the diagram... what is this "operator info" it's a completely undescriptive name and one cannot understand what information is published on this topic. it's fine to not to stick to the names I propose but not like this, when you erase the content that carries important information...

Reply:
I agree. changed 👍